hamilton/parameter_analysis/automation.py

```diff
@@ -35,7 +40,8 @@ def generate_new_param_file(best_params, output_filename, iteration_number):
     for k in best_params.keys():
         if k in valid_keys:
             percentage_change = (
-                CUSTOM_PERCENTAGE_CHANGE.get(k, 1) / 1.5 ** iteration_number
+                CUSTOM_PERCENTAGE_CHANGE.get(k, DEFAULT_PERCENTAGE_CHANGE)
+                / 1.2 ** iteration_number
```

Comment:
Perhaps 1.2 could be assigned to a variable called, say, `learning_rate` so that the amount of change can be easily modified.